plugin-server/tests/utils/event.test.ts

```diff
@@ -134,7 +134,47 @@ describe('formPipelineEvent()', () => {
             $set_once: {},
         })
     })
+    it('forms pluginEvent from a raw message with properties and event at the top level', () => {
```

Comment:
Should we update the tests to use the new structure? And rather add a tests for backwards compatible structure?

Reply:
testing both cases extensively now